python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -607,7 +611,12 @@ def errFunc(p, x, y):
                 assert (len(mask) == len(timeVecOriginal) == len(meanVecOriginal) == len(varVecOriginal))
             if not (mask.any() and newMask.any()):
                 continue
-            dataset.expIdMask[ampName] = mask  # store the final mask
+            dataset.expIdMask[ampName] = np.array(dataset.expIdMask[ampName])
```

Comment:
I'm sure the answer is "yes", but these are mask==1 is good, correct?

Reply:
More precisely, they have `True` (correct) and `False`